src/Agent/NewRelic/Agent/Core/Configuration/DefaultConfiguration.cs

```diff
@@ -1990,6 +1990,20 @@ public virtual bool LogDecoratorEnabled
             }
         }
 
+        private HashSet<string> _logLevelDenyList;
+        public virtual HashSet<string> LogLevelDenyList
+        {
+            get
```

Comment:
Could we log the deny list if we found one? I'm guessing we'll run into some "why doesn't this work" questions where the answer is a typo.

Reply:
@chynesNR I suppose we could, just not sure where we would do that at. Ideas?

Reply:
Here is probably ok, since it only happens once. It looks like we log some similar things in DefaultConfiguration

Reply:
Rather, this is the place we could enforce it only happening once

Reply:
I didn't see the other logging that's already happening in DefaultConfiguration; I'll add it here also.